earthly/flutter/Earthfile

```diff
@@ -0,0 +1,130 @@
+VERSION 0.8
+
+flutter-base:
+    ARG FLUTTER_VERSION=3.22.0
+    ARG WORKDIR=frontend
+
+    FROM debian:bookworm-slim
+
+    RUN apt-get update
+    RUN apt-get install -y git curl unzip bzip2 bash jq gpg lcov
+    COPY --dir scripts .
+    RUN chmod +x scripts/install-chrome-linux64.sh && ./scripts/install-chrome-linux64.sh
+    RUN chmod +x scripts/install-edge-linux64.sh && ./scripts/install-edge-linux64.sh
+    RUN chmod +x scripts/install-firefox-linux64.sh && ./scripts/install-firefox-linux64.sh
+
+    WORKDIR $WORKDIR
+
+    RUN git clone --depth 1 --branch $FLUTTER_VERSION https://github.com/flutter/flutter.git /usr/local/flutter
```

Comment:
@stevenj Seems like the cache doesn't work efficiently, even when using earthly `GIT CLONE`

Reply:
If we make the install scripts executable, and then check that in to git, the executable flag will be saved by git, and we shouldn't need to chmod the scripts.  Its possible the chmod is interfering with the caching.

Reply:
Made scripts executable.